queue_job/jobrunner/runner.py

```diff
@@ -188,7 +188,8 @@ def _datetime_to_epoch(dt):
 
 
 def _odoo_now():
-    dt = datetime.datetime.utcnow()
+    # Need naive datetime
+    dt = datetime.datetime.now(datetime.timezone.utc).replace(tzinfo=None)
```

Comment:
I agree with @sbidoul 

We can remove the funciton `_datetime_to_epoch` and directly return:
```suggestion
    return time.time()
```

I tested localy and in fact, `_odoo_now()` returns a `float`:
```
>>> dt = datetime.datetime.now(datetime.timezone.utc).replace(tzinfo=None)
>>> (dt - datetime.datetime(1970, 1, 1)).total_seconds()
1757943419.659445
>>> time.time()
1757943426.0511284
>>> 
```

Reply:
I agree with @AnizR comment

Reply:
> I agree with @sbidoul
> 
> We can remove the funciton `_datetime_to_epoch` and directly return:
> 
> I tested localy and in fact, `_odoo_now()` returns a `float`:
> 
> ```
> >>> dt = datetime.datetime.now(datetime.timezone.utc).replace(tzinfo=None)
> >>> (dt - datetime.datetime(1970, 1, 1)).total_seconds()
> 1757943419.659445
> >>> time.time()
> 1757943426.0511284
> >>> 
> ```

As discussed with @acsonefho, I have done the refactor of the function `_odoo_now()`.